examples/nixnet/flexray-signal-single-point-input.py

```diff
@@ -100,8 +100,9 @@ def check_for_error(status):
     )
     check_for_error(set_property_response.status)
 
+    # To get more meaningful data use this with output example
```

Comment:
Avoid using 'meaningful'. Rephrase this to:
If no values are being written on the FlexRay port, the signals read would contain the default value of 0.0

Reply:
Done